plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -402,4 +396,22 @@ protected void copyTpchNationToTable(String testTable)
     {
         computeActual(format("INSERT INTO " + testTable + " SELECT name, comment, nationkey, regionkey FROM tpch.tiny.nation"));
     }
+
+    protected void assertFileSizeRange(String testTable, int scaleFactorInThousands, long fileSizeRangeStart, long fileSizeRangeEnd)
```

Comment:
nit: private?

also maybe rename it to `testWriteWithFileSize` or sth in the line (the method is more than just assertion)